library/HTMLPurifier/HTML5Config.php

```diff
@@ -77,6 +80,14 @@ public function __construct(HTMLPurifier_ConfigSchema $schema, HTMLPurifier_Prop
             $schema->add('HTML.Forms', false, 'bool', false);
         }
 
+        if (empty($schema->info['HTML.Link'])) {
+            $schema->add('HTML.Link', array(), 'list', false);
```

Comment:
Why `list`? I'd expect a boolean here. Also this setting does nothing - I'd expect it to be responsible for toggling `HTML5_Link` module.

Reply:
Originally had it accepting an `enum` of allowed `href` values but changed it to `URI.SafeLinkRegexp` so it's more flexible. I'll change it to `boolean` 👍  and update `README.md`